Quick_Deploy/vLLM/README.md

```diff
@@ -84,17 +55,18 @@ and
 [EngineArgs](https://github.com/vllm-project/vllm/blob/32b6816e556f69f1672085a6267e8516bcb8e622/vllm/engine/arg_utils.py#L11)
 for supported key-value pairs.
 
-*Note*: vLLM greedily consume upto 90% of the GPU's memory under default settings. You can provide
+*Note*: vLLM greedily consumes up to 50% of the GPU's memory under default settings. You can tweak this behavior using
```

Comment:
I think this should still be 90%. 
https://github.com/vllm-project/vllm/blob/main/vllm/engine/arg_utils.py#L27

Reply:
Just add a line towards end like...

settings via `vllm_engine_args.json`. For the purpose of the demo model we are setting this to 0.50.

Reply:
Good catch! Updated the wording.